codetransformer/transformers/literals.py

```diff
@@ -42,3 +42,30 @@ def visit_STORE_MAP(self, instr):
 
         yield Instruction(ops.STORE_SUBSCR)
         # TOS  = m
+
+
+class overloaded_bytes(CodeTransformer):
+    """
+    Decorator that applies a callable to each bytes literal in the decorated
+    function.
+
+    Parameters
+    ----------
+    f : callable
+        A callable to be applied to each bytes literal in the decorated
+        function.
+    """
+
+    def __init__(self, f):
+        super().__init__()
+        self.f = f
+
+    def visit_consts(self, consts):
+        return super().visit_consts(
+            tuple(
+                self.visit_consts(const) if isinstance(const, tuple)
```

Comment:
@llllllllll it's possible that recursing through tuples like this should be the default behavior.